localization/autoware_ndt_scan_matcher/include/autoware/ndt_omp/LICENSE

```diff
@@ -0,0 +1,25 @@
+BSD 2-Clause License
```

Comment:
The path for the header file needs to be under `autoware/ndt_scan_matcher` according to our file location guidelines, so it'll be `autoware/ndt_scan_matcher/ndt_omp` if you want to keep `ndt_omp` separate from the rest of the code.

Reply:
Fixed in https://github.com/autowarefoundation/autoware.universe/pull/8912/commits/5bac6f4fa3b17e961766d4bfbf61df121d1b4687

Reply:
I'm sorry, the following include guard becomes too long to pass pre-commit.ci.
https://github.com/autowarefoundation/autoware.universe/blob/4bf27bd60a9c6641acae298974a335657d6a949e/localization/autoware_ndt_scan_matcher/include/autoware/ndt_scan_matcher/ndt_omp/multigrid_pclomp/multi_voxel_grid_covariance_omp_impl.hpp#L52-L53

I will fix again.

Reply:
In fact, given that `ndt_omp` is not going to be exported outside the `ndt_scan_matcher` package, it'd be preferrable to move the `ndt_omp` headers to `src/`

Reply:
Currently, `ndt_omp` includes a static tool to execute NDT for a rosbag. 
e.g.) <https://github.com/tier4/ndt_omp/tree/tier4/main/apps>
It is useful for testing the NDT internal behavior.

After moving the code into `ndt_scan_matcher`, we plan to implement such a tool in `autoware_tools` or somewhere similar. Therefore, we would like to leave the header files as they are, and I have moved the implementation files to src.
https://github.com/autowarefoundation/autoware.universe/pull/8912/commits/6773e83bb070187ff7c46a94f85319d073fcf159

Reply:
@esteve 
I have contacted Dr. Koide and confirmed that this pull request is okay to be merged.
Ando-san has also approved this, so if you approve it, I will merge it.

Reply:
@esteve friendly ping.

Reply:
@esteve friendly ping.

Reply:
@mitsudome-r 
@esteve -san has marked "request changes" on this PR. While I've implemented the requested fixes, I haven't received any further response. Could you force a merge?

Reply:
@SakodaShintaro sorry, I missed your comment. I've approved this PR, thanks for addressing my feedback.